R/cli_messages.R

```diff
@@ -0,0 +1,62 @@
+#' Check if the data argument is an object of class `FIMSFrame`
+#'
+#' @description
+#' This function checks whether the provided `data` argument is an object of the
+#' class `FIMSFrame`. It ensures that the data adheres to the required structure
+#' for use with FIMS-related functions.
+#' @param data The object to check.
+#' @return NULL If the check passes (i.e., `data` is of class `FIMSFrame`).
```

Comment:
The text after @return should start on their own line, and I also provided some edits to the text
```
#' @return
#' If the check passes (i.e., `data` is of class `FIMSFrame`), then `NULL` is returned,
#' otherwise the function is aborted and an error is thrown.